internal/util/tree/tree.go

```diff
@@ -61,69 +63,98 @@ var (
 	cyan   = color.New(color.FgCyan)
 )
 
+// CreateObjectTree creates a new tablewriter.Table for the object tree.
+// Returns a new tablewriter.Table for the object tree.
+func CreateObjectTree(w io.Writer) *tablewriter.Table {
```

Comment:
should this be 
```suggestion
func createObjectTree(w io.Writer) *tablewriter.Table {
```

Same for createObjectTreeV1Beta1